src/checks_gen.rs

```diff
@@ -2,11 +2,13 @@
 
 use colored::Colorize;
 use serde::{Deserialize, Serialize};
-use strum_macros::EnumString;
+use strum_macros::{Display, EnumString};
 
 use crate::checks::CheckCode;
 
-#[derive(EnumString, Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize)]
+#[derive(
+    EnumString, Display, Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Serialize, Deserialize,
```

Comment:
This file is actually auto-generated. Can you add `Display` to the list of derives in `generate_check_code_prefix.rs`? You're looking for this code block:

```rs
// Create the `CheckCodePrefix` definition.
let mut gen = scope
    .new_enum("CheckCodePrefix")
    ...
```

Reply:
Actually, instead of `Display`, can we use `AsRefStr` from `strum_macros`? You'll see that `CheckCode` derives that.

Reply:
Done. Just for my own edification, why is `AsRefStr` better? Or just consistency with `CheckCode`?

Reply:
It's really just for consistency. I believe `AsRefStr` also avoids an allocation: https://github.com/Peternator7/strum/blob/e772e203dabd58630730c1520a2e2110eaf13fc2/strum_macros/src/lib.rs#L125.